release-notes/9.0/supported-os.json

```diff
@@ -0,0 +1,304 @@
+    "last-updated": "2024-05-01",
+    "operating-systems": [
+        {
+            "family": "Android",
+            "distributions": [
+                {
+                    "name": "Android",
+                    "lifecycle-policy": "https://support.google.com/android",
+                    "architectures": [
+                        "x64",
+                        "arm64",
+                        "arm"
+                    ],
+                    "supported-cycles": [
+                        "API 21+"
+                    ]
+                }
+            ]
+        },
+        {
+            "family": "Apple",
+            "distributions": [
+                {
+                    "name": "iOS",
+                    "lifecycle-policy": "https://support.apple.com/ios/",
+                    "architectures": [
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "12.2"
+                    ]
+                },
+                {
+                    "name": "iOS Simulator",
+                    "lifecycle-policy": "https://support.apple.com/ios/",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "12.2",
+                        "13",
+                        "14",
+                        "15",
+                        "16",
+                        "17"
+                    ]
+                },
+                {
+                    "name": "tvOS",
+                    "lifecycle-policy": "https://support.apple.com/apple-tv/",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "12.2",
+                        "13",
+                        "14",
+                        "15",
+                        "16",
+                        "17"
+                    ]
+                },
+                {
+                    "name": "tvOS Simulator",
+                    "lifecycle-policy": "https://support.apple.com/apple-tv/",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "12.2",
+                        "13",
+                        "14",
+                        "15",
+                        "16",
+                        "17"
+                    ]
+                },
+                {
+                    "name": "Mac Catalyst",
+                    "lifecycle-policy": "https://support.apple.com/macos/",
+                    "architectures": [
+                        "x64",
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "15+"
```

Comment:
We can't really sync with iOS here since we support earlier iOS versions than what Mac Catalyst supports, using `15.0+` is correct here.

Reply:
I also think this is mostly because of min-sdk and supported-cycles being mixed up. The min-sdk will be separate of iOS, the supported-cycles most probably not.